warehouse/deltalake/deltalake.go

```diff
@@ -1155,40 +1165,7 @@ func GetDatabricksVersion() (databricksBuildVersion string) {
 	return
 }
 
-// GetDatabricksVersion Gets the databricks version by making a grpc call to Version stub.
```

Comment:
This is not required now. Earlier this was added because we didn't have databricks deployment. Now databricks deployment is by default present.